test/convenience/webhook.test.ts

```diff
@@ -0,0 +1,105 @@
+/// <reference types="npm:@types/node" />
+
+import type { Hono } from "https://deno.land/x/hono/mod.ts";
+import type {
+    APIGatewayProxyEventV2,
+    Context as LambdaContext,
+} from "https://deno.land/x/lambda/mod.ts";
+import type { NHttp } from "https://deno.land/x/nhttp/mod.ts";
+import type { Application } from "https://deno.land/x/oak/mod.ts";
+import type { createServer } from "node:http";
+import type { FastifyInstance } from "npm:fastify";
+import type { NextApiRequest, NextApiResponse } from "npm:next";
+import { Bot, webhookCallback } from "../../src/mod.ts";
+import { describe, it } from "../deps.test.ts";
+
+import { type Express } from "npm:@types/express@^4.17";
+import type Koa from "npm:@types/koa@^2.15";
+import type { UserFromGetMe } from "../../src/types.ts";
+
+describe("webhook", () => {
+    const bot = new Bot("dummy", { botInfo: {} as unknown as UserFromGetMe });
```

Comment:
Just wondering why we need to set `botInfo` here.

Reply:
This avoids the call to `getMe` when updates are handled. Usually, grammY initialises the bot as soon as the first update arrives.

Reply:
But in testing code, the first update would never arrive. Is this for when we eventually decide to test runtime code?

Reply:
The first update does arrive in the tests where we supply the fake updates, such as for NextJS.